org.flathub.flatpak-external-data-checker.yml

```diff
@@ -1,7 +1,7 @@
 id: org.flathub.flatpak-external-data-checker
 sdk: org.freedesktop.Sdk
 runtime: org.freedesktop.Sdk
```

Comment:
Shouldn't it be `org.freedesktop.Platform` or does the external data checkers requires some compilers?

Reply:
It needs git. And since people using f-e-d-c locally to update manifests are probably going to build those manifests, they likely to have the SDK installed anyway.

Reply:
👍